atlasdb-conjure/src/main/java/com/palantir/atlasdb/http/v2/ClientOptions.java

```diff
@@ -129,4 +124,27 @@ public static ClientOptions of(Duration connect, Duration read, Duration backoff
                 .maxNumRetries(retries)
                 .build();
     }
+
+    static ClientOptions fromRemotingParameters(AuxiliaryRemotingParameters parameters) {
+        ImmutableClientOptions.Builder builder = ImmutableClientOptions.builder();
+
+        setupTimeouts(builder, parameters);
+        setupRetrying(builder, parameters);
+
+        return builder.clientQoS(ClientConfiguration.ClientQoS.DANGEROUS_DISABLE_SYMPATHETIC_CLIENT_QOS)
+                .build();
+    }
+
+    private static void setupTimeouts(ImmutableClientOptions.Builder builder, AuxiliaryRemotingParameters parameters) {
```

Comment:
some docs would be good, to describe what the effect each combination will have, so we don't have to do the same exercise we did today, or even if we do, it's a lot easier.

either here or the constants above.

Reply:
Added a bit of explanation - I'm not sure exactly what you want, might be worth catching up if there are more specific things you'd like to be documented. Thanks!

Reply:
This'll do, I suppose I wanted a way to codify our investigation, something along the lines of, anything below a proxy probably shouldn't retry etc. What the current setup is, if there are separate timelock to timelock configurations vs client to timelock etc.